cypress/integration/EnrollmentEditEventPage/EnrollmentEditEventPageForm.feature

```diff
@@ -73,14 +73,14 @@ And the user see the following text: 11
     When the user clicks on the cancel button
     Then the user see the following text: Enrollment Dashboard
 
-Scenario: User can update schedule date for a scheduled event 
-    Given you land on the enrollment event page with selected person by having typed /#/enrollmentEventEdit?eventId=RIrfCcEP8Uu&orgUnitId=DiszpKrYNg8
-    Then the user see the following text: Enrollment: Edit Event
-    And the user see the following text: Infant Feeding
-    When the user clicks switch tab to Schedule
-    And the user selects another schedule date
-    And the user clicks on the schedule button on widget-enrollment-event
-    Then the user see the following text: Enrollment Dashboard
+# Scenario: User can update schedule date for a scheduled event 
```

Comment:
Hey, any reason why this test was disabled?

Reply:
Not any more, so good catch 👍
We had this problem here 3 weeks ago: https://dhis2.atlassian.net/browse/DHIS2-14301. Will readd test.